### DISCUSSION TITLE
Bring back Following

### DISCUSSION THREAD

**Discussion by Codel1417** [community/community#37537] [Other Feature Feedback, Questions, & Ideas]
The new feed is useless curated junk with an algorithm worse than reddit search. How did this even make it to testing or production?

Please bring back the old following tab sorted by date, without any curation or algorithm.

**Comment by vi** [comment]
Now using a hacky workaround using Github API and bunch of shell scripts: https://gist.github.com/vi/5b965253c869422a6bad32ad57f05790

It download all metadata of all repositories, so is slow.
